From 845faa55b88079f4db3fc9f8c5890e611b92f5f3 Mon Sep 17 00:00:00 2001
From: "A.I" <ailis@paw.zone>
Date: Thu, 3 Aug 2017 12:03:27 -0700
Subject: [PATCH] ncurses: drop an unneeded trailing backslash

---
 package/ncurses/ncurses.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package/ncurses/ncurses.mk b/package/ncurses/ncurses.mk
index 20c82c05fe..ad7f802287 100644
--- a/package/ncurses/ncurses.mk
+++ b/package/ncurses/ncurses.mk
@@ -67,7 +67,7 @@ NCURSES_TERMINFO_FILES = \
 	x/xterm+256color \
 	x/xterm-256color \
 	x/xterm-color \
-	x/xterm-xfree86 \
+	x/xterm-xfree86
 
 ifeq ($(BR2_PACKAGE_NCURSES_WCHAR),y)
 NCURSES_CONF_OPTS += --enable-widec
-- 
2.14.1

