From c0795808eeaeba66e4b72a55e40ad9daa7847b8e Mon Sep 17 00:00:00 2001
From: "A.I" <ailis@paw.zone>
Date: Thu, 3 Aug 2017 12:03:27 -0700
Subject: [PATCH] ncurses: drop an unneeded trailing backslash

---
 package/ncurses/ncurses.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package/ncurses/ncurses.mk b/package/ncurses/ncurses.mk
index 952be566e8..cc3f81b107 100644
--- a/package/ncurses/ncurses.mk
+++ b/package/ncurses/ncurses.mk
@@ -64,7 +64,7 @@ NCURSES_TERMINFO_FILES = \
 	v/vt220 \
 	x/xterm \
 	x/xterm-color \
-	x/xterm-xfree86 \
+	x/xterm-xfree86
 
 ifeq ($(BR2_PACKAGE_NCURSES_WCHAR),y)
 NCURSES_CONF_OPTS += --enable-widec
-- 
2.14.1

