From a94739329396cf4405f19ee778fa4e8a7da05160 Mon Sep 17 00:00:00 2001
From: "A.I" <ailis@paw.zone>
Date: Mon, 15 May 2017 15:52:29 -0700
Subject: [PATCH] openssl: Remove debug information to reduce its footprint

---
 package/openssl/openssl.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/package/openssl/openssl.mk b/package/openssl/openssl.mk
index ec8e6f204c..199fd7d39d 100644
--- a/package/openssl/openssl.mk
+++ b/package/openssl/openssl.mk
@@ -87,6 +87,8 @@ define OPENSSL_CONFIGURE_CMDS
 			enable-tlsext \
 			$(if $(BR2_STATIC_LIBS),zlib,zlib-dynamic) \
 			$(if $(BR2_STATIC_LIBS),no-dso) \
+			no-err \
+			no-filenames \
 	)
 	$(SED) "s#-march=[-a-z0-9] ##" -e "s#-mcpu=[-a-z0-9] ##g" $(@D)/Makefile
 	$(SED) "s#-O[0-9]#$(OPENSSL_CFLAGS)#" $(@D)/Makefile
-- 
2.14.1

