From 220317c05c7376ebc5efa96972ac6be893c9de36 Mon Sep 17 00:00:00 2001
From: "A.I" <ailis@paw.zone>
Date: Tue, 19 Feb 2019 17:04:30 -0800
Subject: [PATCH] package/libopenssl: Add OPENSSL_SMALL_FOOTPRINT

---
 package/libopenssl/libopenssl.mk | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/package/libopenssl/libopenssl.mk b/package/libopenssl/libopenssl.mk
index a6b97a0957..783569e28b 100644
--- a/package/libopenssl/libopenssl.mk
+++ b/package/libopenssl/libopenssl.mk
@@ -20,8 +20,9 @@ ifeq ($(BR2_m68k_cf),y)
 # relocation truncated to fit: R_68K_GOT16O
 LIBOPENSSL_CFLAGS += -mxgot
 # resolves an assembler "out of range error" with blake2 and sha512 algorithms
-LIBOPENSSL_CFLAGS += -DOPENSSL_SMALL_FOOTPRINT
+# LIBOPENSSL_CFLAGS += -DOPENSSL_SMALL_FOOTPRINT
 endif
+LIBOPENSSL_CFLAGS += -DOPENSSL_SMALL_FOOTPRINT
 
 ifeq ($(BR2_USE_MMU),)
 LIBOPENSSL_CFLAGS += -DHAVE_FORK=0 -DOPENSSL_NO_MADVISE
-- 
2.17.2 (Apple Git-113)

