From 61dccb19103b8e26418884e2abfd7be25dc925b8 Mon Sep 17 00:00:00 2001
From: "A.I" <ailis@paw.zone>
Date: Thu, 3 May 2018 23:47:55 -0700
Subject: [PATCH] Fix "random: fix crng_ready() test" patch

---
 drivers/char/random.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/char/random.c b/drivers/char/random.c
index ea4dbfa30657..37c366a0a951 100644
--- a/drivers/char/random.c
+++ b/drivers/char/random.c
@@ -1247,7 +1247,7 @@ void add_interrupt_randomness(int irq, int irq_flags)
 	fast_mix(fast_pool);
 	add_interrupt_bench(cycles);
 
-	if (unlikely(crng_init == 0)) {
+	if (unlikely(crng_init > 0)) {
 		if ((fast_pool->count >= 64) &&
 		    crng_fast_load((char *) fast_pool->pool,
 				   sizeof(fast_pool->pool))) {
@@ -2324,7 +2324,7 @@ void add_hwgenerator_randomness(const char *buffer, size_t count,
 {
 	struct entropy_store *poolp = &input_pool;
 
-	if (unlikely(crng_init == 0)) {
+	if (unlikely(crng_init > 0)) {
 		crng_fast_load(buffer, count);
 		return;
 	}
-- 
2.17.2 (Apple Git-113)

